src/poliastro/twobody/events.py

```diff
@@ -80,3 +87,80 @@ class LithobrakeEvent(AltitudeCrossEvent):
 
     def __init__(self, R, terminal=True):
         super().__init__(0, R, terminal, direction=-1)
+
+
+class LatitudeCrossEvent(Event):
+    """Detect if a satellite crosses a specific threshold latitude.
+
+    Parameters
+    ----------
+    lat: float
+        Threshold latitude (km).
+    R: float
+        Radius of the attractor (km).
+    terminal: bool
+        Whether to terminate integration if this event occurs, defaults to True.
+    direction: float
+        Handle triggering of event based on whether latitude is crossed from above
+        or below, defaults to 0, i.e., event is triggered while traversing from both directions.
+
+    """
+
+    def __init__(self, lat, R, terminal=True, direction=0):
```

Comment:
As discussed in the weekly meeting on July the 2nd, let us pass a `~poliastro.bodies.body`, so we can check if body is the `Earth` and raise a `NotImplementedError` till #1269 is better discussed.

Reply:
In fact, let us pass the orbit to be propagated so we can compute:

```python
self._R = orbit.attractor.R
self._epoch0 = orbit.epoch
```
those will be useful for later logic.